vitest.config.ts

```diff
@@ -1,27 +1,36 @@
 import tsconfigPaths from "vite-tsconfig-paths";
-import { defineConfig } from "vitest/config";
+import { configDefaults, defineConfig } from "vitest/config";
 
 export default defineConfig({
 	plugins: [tsconfigPaths()],
 	test: {
+		environment: "node",
+		include: [
+			"src/**/*.test.{ts,tsx,js,jsx}",
+			"src/**/*.spec.{ts,tsx,js,jsx}",
+			"test/**/*.test.{ts,tsx,js,jsx}",
+			"test/**/*.spec.{ts,tsx,js,jsx}",
+		],
+		exclude: [
+			...configDefaults.exclude,
+			"dist/**",
+			"coverage/**",
+			"docs/**",
+			"**/*.d.ts",
+		],
+		hookTimeout: 30000,
+		testTimeout: 60000,
+		passWithNoTests: false,
 		coverage: {
-			provider: "v8", // or 'istanbul' if you prefer
-			reporter: ["text", "lcov", "html"],
+			provider: "v8",
+			reporter: ["text", "html", "lcov"],
+			exclude: [
+				...(configDefaults.coverage?.exclude ?? []),
+				"dist/**",
+				"coverage/**",
+				"docs/**",
+				"**/*.d.ts",
+			],
 		},
-		// https://vitest.dev/config/#fileparallelism
-		// fileParallelism: true,
-
```

Comment:
All these deleted lines must be restored. You are impacting the known working performance of our testing regime.

<img width="916" height="447" alt="Image" src="https://github.com/user-attachments/assets/421786b7-d21a-4d82-877c-86cb8ea807ab" />